crates/common/src/commit/request.rs

```diff
@@ -32,19 +43,28 @@ impl SignedProxyDelegation {
     }
 }
 
+impl fmt::Display for SignedProxyDelegation {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}\nSignature: {}", self.message, self.signature)
+    }
+}
+
+// TODO(David): Consider splitting `SignRequest` into two: ConsensusSignRequest
```

Comment:
Split this up.

Reply:
Resolved in [eba97b0](https://github.com/Commit-Boost/commit-boost-client/pull/87/commits/eba97b0ca03b140bbfdc9f4b7b8ad4adf382776b)